nuspec/netcoreglobaltool/dotnet-sonarscanner.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
   <metadata>
     <id>dotnet-sonarscanner</id>
-    <version>9.0.0</version>
```

Comment:
This is how choco and nuget deduce that this is a pre-release.
See [here](https://learn.microsoft.com/en-us/nuget/create-packages/prerelease-packages#installing-and-updating-pre-release-packages).